.github/stale.yml

```diff
@@ -8,10 +8,9 @@ daysUntilClose: 30
 
 # Issues with these labels will never be considered stale
 exemptLabels:
-  - pinned
-  - security
   - accessibility
-  - "breaking change :boom:"
+  - "breaking change"
+  - design approved
```

Comment:
do we need quotes? `"breaking change"` has quotes but `design approved` doesn't.

Reply:
There should not be quotes - removed these